tools/osx/xcode_configure.bzl

```diff
@@ -295,6 +295,9 @@ def _impl(repository_ctx):
     repository_ctx.file("BUILD", build_contents)
 
 xcode_autoconf = repository_rule(
+    environ = [
+        "XCODE_VERSION",
```

Comment:
I'll add `DEVELOPER_DIR` here as well, since it influences the default Xcode version selected.